packages/elements/package.json

```diff
@@ -334,6 +334,7 @@
     "chart.js": "~2.9.4",
     "d3-interpolate": "^3.0.1",
     "date-fns": "^2.22.1",
+    "idb": "^7.0.2",
```

Comment:
Should this be added to `utils` package.json?

Reply:
Updated. And also removed from @refinitiv-ui/elements.